docs/schema/records_reference.md

```diff
@@ -1,6 +1,6 @@
 # Record Reference
 
-Whereas there can be multiple releases about a contracting process, there should be a single **record** per contracting process, aggregating all the releases available for the contracting process.
+A record aggregates the releases related to a contracting process. There should be a single record per contracting process per [distribution](https://www.w3.org/TR/vocab-dcat-2/#Class:Distribution), where a distribution might be a specific API endpoint or a specific bulk download file.
```

Comment:
I prefer using "releases **about** a contracting process" throughout instead of **related to**, because a release is a data artifact that describes a real-world contracting process (an "about" relationship), whereas "relates to" is much more generic (e.g. a release about a contract renewal could, in a sense, "relate" to the original contract).

Reply:
Makes sense. Fixed in 5b055f71e421eec4796da49c07a827fcc7517162